arcus_orientation.md

```diff
@@ -697,7 +698,7 @@ The [training videos](#training-videos) walk through everything you need to get
 
 ## Security Feature: Internet Disabled Mode
 
-While there are myriad benefits to taking a scripted approach to data analysis, it does also introduce some specific privacy and security risks. Thankfully, Arcus has taken these risks into consideration, and developed our [Internet Disabled Mode](https://forum.arcus.chop.edu/t/about-the-arcus-lab-security-setting-for-disabling-internet-in-labs/741) feature. The feature provides peace of mind by allowing you to work on your code in an environment that is entirely disconnected from the internet, keeping your data and code safer. 
+While there are myriad benefits to taking a scripted approach to data analysis, it does also introduce some specific privacy and security risks. Thankfully, Arcus has taken these risks into consideration, and developed our [Internet Disabled mode](https://forum.arcus.chop.edu/t/about-the-arcus-lab-security-setting-for-disabling-internet-in-labs/741) feature. The feature provides peace of mind by allowing you to work on your code in an environment that is entirely disconnected from the internet, keeping your data and code safer. 
```

Comment:
How do we feel about dropping the word "Thankfully" here? It might just be my midwestern roots showing, but I feel weird expressing gratitude for ourselves... @franzenr

Reply:
I'm totally fine dropping that word! It existed mainly to help _me_ transition between the ideas in my mind while writing, but I also felt a little weird about it. @rosemm

Reply:
What about...

```suggestion
While there are myriad benefits to taking a scripted approach to data analysis, it does also introduce some specific privacy and security risks. To mitigate these risks, Arcus has developed our [Internet Disabled mode](https://forum.arcus.chop.edu/t/about-the-arcus-lab-security-setting-for-disabling-internet-in-labs/741) feature. The feature provides peace of mind by allowing you to work on your code in an environment that is entirely disconnected from the internet, keeping your data and code safer. 
```

Reply:
```suggestion
While there are myriad benefits to taking a scripted approach to data analysis, it does also introduce some specific privacy and security risks. Arcus has taken these risks into consideration, and developed our [Internet Disabled mode](https://forum.arcus.chop.edu/t/about-the-arcus-lab-security-setting-for-disabling-internet-in-labs/741) feature for labs. The feature provides peace of mind by allowing you to work on your code in an environment that is entirely disconnected from the internet, keeping your data and code safer. 
```